src/proof_system/widget/mod.rs

```diff
@@ -143,15 +160,16 @@ impl VerifierKey {
             arithmetic,
             logic,
             range,
-            ecc,
+            variable_base,
+            fixed_base,
             permutation,
         };
         Ok(verifier_key)
     }
 
     const fn serialised_size() -> usize {
         const N_SIZE: usize = 8;
-        const NUM_COMMITMENTS: usize = 14;
+        const NUM_COMMITMENTS: usize = 15;
```

Comment:
We appended two more commitments, but increased the const to `15` instead of `16`. Why?

Reply:
This PR only adds one new custom gate, this new custom gate allows for a more efficient way to do variable base scalar multiplication